### ISSUE TITLE
Uuid is not supported in oapi::endpoint macro

### ISSUE THREAD

**Issue by Roba1993** [salvo-rs/salvo#414]
**Describe the bug**
ToSchema seems not to be implemented completly for Uuid.

**To Reproduce**
My `Cargo.toml` configuration:
```toml
[dependencies]
salvo = { version = "0.55", features = ["rustls", "serve-static", "affix", "websocket", "compression", "anyhow", "session"] }
salvo-oapi = { version = "0.55", features = ["swagger-ui", "rapidoc", "uuid", "time"] }
uuid = { version = "1.2", features = ["v4", "serde"] }
```

How I use the `#handle` macro:
```rust
#[endpoint]
pub async fn get_record_rest(
    env_uuid: HeaderParam<Uuid, true>,
) -> Json<String> {
    Json("test".to_string())
}
```

**Expected behavior**
The Uuid typ should be supported.

**Comment by chrislearn** [comment]
Have you enable serde feature in Cargo.toml?

```toml
[dependencies]
uuid = {  workspace = true, features = ["serde"] }
```

**Comment by Roba1993** [comment]
Yes I have, sorry I have not put all dependencies into the example...

**Comment by chrislearn** [comment]
fixed in version: v0.55.3
